metecho/api/models.py

```diff
@@ -1067,6 +1067,21 @@ def finalize_provision(self, *, error=None, originating_user_id):
             else:
                 self.delete(originating_user_id=originating_user_id)
 
+    def queue_convert_to_dev_org(self, task, *, originating_user_id=None):
+        from .jobs import convert_to_dev_org_job
+
+        convert_to_dev_org_job.delay(
+            scratch_org=self, task=task, originating_user_id=originating_user_id
+        )
+
+    def finalize_convert_to_dev_org(self, task, *, originating_user_id):
+        self.org_type = SCRATCH_ORG_TYPES.Dev
+        self.task = task
+        self.epic = None
+        self.save()
+        task.notify_changed(originating_user_id=originating_user_id)
+        self.notify_changed(originating_user_id=originating_user_id)
```

Comment:
I think this should tell the frontend that the task and scratch org have changed, but let me know if I should be using some other `notify_*` method. I tried a couple but never got WS messages to show up, perhaps because I was triggering the changes to the scratch org separately via DRF?

Reply:
Yeah, this is all that would be needed, but I noticed like you that it wasn't firing as expected. Upon further digging, I discovered that the `scratch_org.id` used here was different than the same `scratch_org.id` shown in the API -- so the WS message was being sent, but to a different channel than what the front end was subscribed to. I'm not 100% sure why that happens (tho it sounds familiar) -- I think related to different hashes between the Django process and the RQ process? Regardless, adding a `refresh_from_db` in the job seems to fix it.